examples/_empty/src/experiment.py

```diff
@@ -0,0 +1,43 @@
+from collections import OrderedDict
+import torchvision
+from torchvision import transforms
+from catalyst.dl import ConfigExperiment
+
+
+class Experiment(ConfigExperiment):
+    @staticmethod
+    def get_transforms(stage: str = None, mode: str = None):
+
+        # CHANGE ME
+        result = transforms.Compose(
+            [
+                transforms.ToTensor(),
+                transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+            ]
+        )
+
+        return result
+
+    def get_datasets(self, stage: str, **kwargs):
```

Comment:
it would be nice to add here some arguments to give users an intuition about `data_params` from config